windows/NsisInclude/NsisMultiUserLang.nsh

```diff
@@ -996,68 +1781,98 @@ Version: 2023-01-30
 	; text string for installer section
-	LangString SECTION_CORE_NAME ${LANG_ITALIAN} "Programma (richiesto)"
-	LangString SECTION_CORE_DESCRIPTION ${LANG_ITALIAN} "File principali del programma."
+	LangString SECTION_CORE_NAME ${LANG_ITALIAN} "File programma (richiesti)"
+	LangString SECTION_CORE_DESCRIPTION ${LANG_ITALIAN} "File del programma principale (richiesti)."
 	LangString SECTION_SHORTCUTS_NAME ${LANG_ITALIAN} "Collegamenti"
-	LangString SECTION_SHORTCUTS_DESCRIPTION ${LANG_ITALIAN} "Crea collegamenti."
+	LangString SECTION_SHORTCUTS_DESCRIPTION ${LANG_ITALIAN} "Opzioni creazione creazione collegamenti."
 	LangString SECTION_START_MENU_NAME ${LANG_ITALIAN} "Menu Start"
 	LangString SECTION_START_MENU_DESCRIPTION ${LANG_ITALIAN} "Crea un collegamento nel menu Start."
-	LangString SECTION_DESKTOP_NAME ${LANG_ITALIAN} "Dsktop"
+	LangString SECTION_DESKTOP_NAME ${LANG_ITALIAN} "Desktop"
 	LangString SECTION_DESKTOP_DESCRIPTION ${LANG_ITALIAN} "Crea un collegamento sul desktop."
 	LangString SECTION_QUICK_LAUNCH_NAME ${LANG_ITALIAN} "Avvio rapido"
 	LangString SECTION_QUICK_LAUNCH_DESCRIPTION ${LANG_ITALIAN} "Crea un collegamento in Avvio rapido."
-	LangString SECTION_TRANSLATIONS_NAME ${LANG_ITALIAN} "File lingua interfaccia"
-	LangString SECTION_TRANSLATIONS_DESCRIPTION ${LANG_ITALIAN} "Installa i file lingua interfaccia."
+	LangString SECTION_TRANSLATIONS_NAME ${LANG_ITALIAN} "File traduzione interfaccia"
+	LangString SECTION_TRANSLATIONS_DESCRIPTION ${LANG_ITALIAN} "Installa i file traduzione interfaccia."
 	LangString SECTION_INTEGRATE_SHRED_NAME ${LANG_ITALIAN} "Modulo sovrascrittura"
 	LangString SECTION_INTEGRATE_SHRED_DESCRIPTION ${LANG_ITALIAN} "Installa il modulo sovrascrittura file integrato."
 	LangString SECTION_UNINSTALL_NAME ${LANG_ITALIAN} "Modulo disinstallazione"
-	LangString SECTION_UNINSTALL_DESCRIPTION ${LANG_ITALIAN} "Include il programma per la disinstallazione."
-	LangString NAME_UNINSTALL_TEXT ${LANG_ITALIAN} " ${prodname} verrà disinstallato dalla seguente cartella.$\r$\nFai clic su 'Disinstalla' per avviare la disinstallazione.$\r$\nAVVISO: il programma di disinstallazione rimuove completamente la cartella di installazione, inclusi tutti i file (come i programmi di pulizia personalizzati) eventualmente aggiunti o modificati."
-	LangString ALREADY_INSTALLED ${LANG_ITALIAN} "${prodname} è già installato.$\r$\nFai clic su 'OK' per disinstallare prima la vecchia versione, o fai clic su 'Annulla' per interrompere l'aggiornamento."
+	LangString SECTION_UNINSTALL_DESCRIPTION ${LANG_ITALIAN} "Installa il modulo per la disinstallazione del programma."
+	LangString NAME_UNINSTALL_TEXT ${LANG_ITALIAN} "${prodname} verrà disinstallato dalla seguente cartella.$\r$\nSeleziona 'Disinstalla' per avviare la disinstallazione.$\r$\nAVVISO: il programma di disinstallazione rimuove completamente la cartella di installazione, inclusi tutti i file (come i programmi di pulizia personalizzati) eventualmente aggiunti o modificati."
```

Comment:
NAME_UNINSTALL_TEXT should be UNINSTALL_TEXT

Reply:
Maybe related to this, but the component description in the variable SECTION_UNINSTALL_DESCRIPTION during uninstall is not displayed.

![image](https://user-images.githubusercontent.com/3830310/221792522-936fcb79-c8eb-4344-8c95-fba3d73eda21.png)

Reply:
I already prepared the strings (that it's a big job) to enable it.
Then I should make other isnpection about UNINSTALl module.
But could require long time.
Then I prefer for this step to leave it in english but have all other strings translated.